### PR TITLE
maint: updating cicd and `start_mapdl` files

### PR DESCRIPTION
**Pull request by clatapie**
As title says.

### CODE REVIEW

**Comment by RobPasMue** [comment]
As I mentioned in https://github.com/ansys/pymapdl-examples/pull/178 it is an issue inside the container

When running in PyMAPDL (if I am not mistaken) - you run things inside the container directly and you install these libraries inside it as well. However, here, you are connecting to it.

**Comment by germa89** [comment]
> As I mentioned in [ansys/pymapdl-examples#178](https://github.com/ansys/pymapdl-examples/pull/178) it is an issue inside the container
> 
> When running in PyMAPDL (if I am not mistaken) - you run things inside the container directly and you install these libraries inside it as well. However, here, you are connecting to it. So whenever you perform the install, it is not being installed for the container AFAICT.

The container hasn't changed in ages (9 months)... why now it is asking for a new library to run? It does not make sense...

**Comment by germa89** [comment]
Opting for avoiding the issue all together....

Let's use the new docker images (do not have this issue).

**Comment by germa89** [comment]
Fixed, images are being pulled and launched properly. https://github.com/ansys/pyansys-math/actions/runs/6956135863?pr=223

There are some tests not passing properly. I leave it up to you @clatapie

**Comment by clatapie** [comment]
The unit test `test_solve_eigs_km` was failing because of a change in the MAPDL output when version > 23.2

Output for version 23.2:
```
*****  INDEX OF DATA SETS ON RESULTS FILE  *****\n\n   SET   TIME/FREQ    LOAD STEP   SUBSTEP  CUMULATIVE\n     1  1475.1             1         1         1                  \n     2  1475.1             1         2         2                  \n     3  2018.8             1         3         3                  \n     4  2018.8             1         4         4                  \n     5  2018.8             1         5         5                  \n     6  2024.8             1         6         6                  \n     7  2024.8             1         7         7                  \n     8  2024.8             1         8         8                  \n     9  2242.2             1         9         9                  \n    10  2274.8             1        10        10
```

Output for version 24.1:
```
*****  INDEX OF DATA SETS ON RESULTS FILE  *****\n\n     SET   TIME/FREQ    LOAD STEP   SUBSTEP  CUMULATIVE\n       1  1475.1333             1         1         1                  \n       2  1475.1333             1         2         2                  \n       3  2018.8374             1         3         3                  \n       4  2018.8374             1         4         4                  \n       5  2018.8374             1         5         5                  \n       6  2024.7868             1         6         6                  \n       7  2024.7868             1         7         7                  \n       8  2024.7868             1         8         8                  \n       9  2242.2153             1         9         9                  \n      10  2274.8300             1        10        10
```

The number of digits after the decimal point increased from 1 (for `23.2`) to 4 ( for`24.1`). The number were therefore no longer caught by the `re.findall` call.

File: doc/source/conf.py [hunk 1]
```diff
@@ -6,7 +6,8 @@
 from ansys_sphinx_theme import ansys_favicon, get_version_match, pyansys_logo_black
 import numpy as np
 import pyvista
-from pyvista.plotting.utilities.sphinx_gallery import DynamicScraper
+
+# from pyvista.plotting.utilities.sphinx_gallery import DynamicScraper
```
Comment (RobPasMue) [inline doc/source/conf.py:10]:
```suggestion
```

**Review by RobPasMue — APPROVED** [review]

**Comment by RobPasMue** [comment]
Ready to merge?

File: .github/workflows/ci_cd.yml [hunk 2]
```diff
@@ -14,20 +14,20 @@ env:
...
-  RESET_PIP_CACHE: 3
+  RESET_AUTOSUMMARY_CACHE: 9
+  RESET_EXAMPLES_CACHE: 9
+  RESET_DOC_BUILD_CACHE: 9
+  RESET_PIP_CACHE: 9
 
 concurrency:
   group: ${{ github.workflow }}-${{ github.ref }}
-  cancel-in-progress: true
+  cancel-in-progress: false # to be moodified
```
Comment (clatapie) [inline .github/workflows/ci_cd.yml:30]:
```suggestion
  cancel-in-progress: true
```

File: .github/workflows/ci_cd.yml [hunk 3]
```diff
@@ -14,20 +14,20 @@ env:
...
   PYMAPDL_PORT: 21000  # default won't work on GitHub runners
   PYMAPDL_DB_PORT: 21001  # default won't work on GitHub runners
-  RESET_AUTOSUMMARY_CACHE: 3
-  RESET_EXAMPLES_CACHE: 3
-  RESET_DOC_BUILD_CACHE: 3
-  RESET_PIP_CACHE: 3
+  RESET_AUTOSUMMARY_CACHE: 9
+  RESET_EXAMPLES_CACHE: 9
+  RESET_DOC_BUILD_CACHE: 9
+  RESET_PIP_CACHE: 9
```
Comment (clatapie) [inline .github/workflows/ci_cd.yml:26]:
```suggestion
  RESET_AUTOSUMMARY_CACHE: 0
  RESET_EXAMPLES_CACHE: 0
  RESET_DOC_BUILD_CACHE: 0
  RESET_PIP_CACHE: 0
```

**Pull request merged**